### PR TITLE
Improve README and add citation guide

### LINKED ISSUE #150: Improve README and add citation guide
Before making a broader public announcement, we need to verify that the correct installation instructions are in place and that a clear citation guide is available pointing to the correct model citations.

### PR DESCRIPTION
**Pull request by afedynitch**
Fixes #150

This is a cleanup of the README. The developer docs have been moved to [doc/dev_docs.md](doc/dev_docs.md) to make more room for user-facing docs.

ToDo:

- [x] CLI docs
- [x] Correct list of formats
- [x] COPYING files with licenses don't exist everywhere

### CODE REVIEW

**Comment by afedynitch** [comment]
@HDembinski, could you please add a few lines to the CLI short doc in the readme? Also, please list the supported formats and how to switch between these.

File: README.md [hunk 1]
```diff
@@ -34,131 +38,55 @@ average_pt = average_pt / nevents
...
-This takes a while. The command installs the package in editable mode (for developing the Python layer) and with verbose output, so that you can watch the compilation happening. Warnings can be ignored, but watch out for errors. Pip automatically creates a virtual environment and downloads dependencies for the build, prefering to install binary wheels of older versions of dependencies if the most recent version has no binary wheel, so you don't have to compile dependencies as well.
-
-To run the tests or try the examples, use this modified `pip install` instead:
+Further examples, such as [this](examples/compare_models.ipynb) can be found in the examples folder.
 
-    pip install --prefer-binary -v -e .'[test,examples]'
+### Command line user interface (CLI) 
 
-which installes chromo and additional optional Python packages that are used in the tests and examples, but not required to run chromo.
+CLI Via a HEPMC output that can be piped in Rivet or other tools supporting the format.
```
Comment (HDembinski) [inline README.md:45]:
```suggestion
CLI via a HEPMC output that can be piped in Rivet or other tools supporting the format.
```

File: README.md [hunk 2]
```diff
@@ -34,131 +38,55 @@ average_pt = average_pt / nevents
...
 
-#### For developers
+## Output formats
 
-If you want to work on the Fortran sources, it is more convenient to install a development version of Chromo with setuptools.
+- plain Python `event` objects
+- HEPMC
```
Comment (HDembinski) [inline README.md:50]:
```suggestion
- HEPMC (optionally gzip compressed)
```

File: README.md [hunk 3]
```diff
@@ -34,131 +38,55 @@ average_pt = average_pt / nevents
...
+- ROOT (via uproot)
+- ..gzip compressed/uncompressed
```
Comment (HDembinski) [inline README.md:52]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -34,131 +38,55 @@ average_pt = average_pt / nevents
...
+- event views via SVG
```
Comment (HDembinski) [inline README.md:53]:
```suggestion
- event views via SVG (via pyhepmc package)
```

Comment (HDembinski) [reply README.md:53]: I am not really sure about about this one, feel free to ignore this.

**Review by HDembinski — APPROVED** [review]
Looks very good from my side, I like the table especially.

In the citation section, we could perhaps already mention that there will be a PoS proceeding soon to cite for Chromo.

File: README.md [hunk 5]
```diff
@@ -34,131 +38,55 @@ average_pt = average_pt / nevents
...
-    cd chromo
-    pip install --prefer-binary -v -e .
-
-This takes a while. The command installs the package in editable mode (for developing the Python layer) and with verbose output, so that you can watch the compilation happening. Warnings can be ignored, but watch out for errors. Pip automatically creates a virtual environment and downloads dependencies for the build, prefering to install binary wheels of older versions of dependencies if the most recent version has no binary wheel, so you don't have to compile dependencies as well.
-
-To run the tests or try the examples, use this modified `pip install` instead:
+Further examples, such as [this](examples/compare_models.ipynb) can be found in the examples folder.
 
-    pip install --prefer-binary -v -e .'[test,examples]'
+### Command line user interface (CLI)
```
Comment (HDembinski) [inline README.md:44]:
```suggestion

Installing `chromo` also makes a command-line interface available. If your Python runtime environment is properly set up, you can do

     chromo --help

To see the help of the command-line interface. If that does not work, just replace `chromo` with `python -m chromo`. The command-line interface was designed to feel familiar for users of [CRMC](https://gitlab.iap.kit.edu/AirShowerPhysics/crmc). The CLI can write events in a variety of output formats, as detailed below.
```

**Comment by afedynitch** [comment]
Thanks for the quick follow up. With the "plain events" I meant the python interface and the direct analysis of these in python. And I agree with you with the way how it was written it was misleading, so removing is the right thing to do. Merging..

**Pull request merged**